server/datastore/mysql/migrations/tables/20240801115359_AddPlatformToHostVPPSoftwareInstalls_test.go

```diff
@@ -0,0 +1,68 @@
+package tables
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUp_20240801115359(t *testing.T) {
+	db := applyUpToPrev(t)
+
+	// Create user
+	u1 := execNoErrLastID(t, db, `INSERT INTO users (name, email, password, salt) VALUES (?, ?, ?, ?)`, "u1", "u1@b.c", "1234", "salt")
```

Comment:
This needed?

Reply:
Yes, `host_vpp_software_installs` needs a user id -- who installed the app.